standalone-server/pom.xml

```diff
@@ -23,8 +23,8 @@ limitations under the License.
     <version>1.10.0-SNAPSHOT</version>
   </parent>
   <artifactId>avatica-standalone-server</artifactId>
-  <name>Avatica Standalone Server</name>
-  <description>A Stadnalone Avatica Server Implementation</description>
+  <name>Apache Calcite Avatica Standalone Server</name>
+  <description>A Standalone Avatica Server Implementation</description>
```

Comment:
The order of the title and description is different? Also not consistent on Apache Calcite being included?